packages/base-driver/lib/basedriver/capabilities.js

```diff
@@ -221,8 +225,30 @@ function processCapabilities (caps, constraints = {}, shouldValidateCaps = true)
   return matchedCaps;
 }
 
+function promoteAppiumOptions (originalCaps) {
```

Comment:
it would be nice to have a docstring here. also, to mention that the function call mutates capabilities

Reply:
ok to the docstring. it doesn't mutate capabilities though?